microsoft-edge/devtools-guide-chromium/css/reference.md

```diff
@@ -426,6 +424,67 @@ To open the **Angle Clock**:
 
 
+<!-- ====================================================================== -->
+## Change box and text shadows with the Shadow Editor
+
+Use the **Shadow Editor** to change the value of the `box-shadow` or `text-shadow` CSS property on an HTML element:
+
+1. [Select an element](#select-an-element) with a `box-shadow` or `text-shadow` declaration.
+
+   For example, open [the 1DIV demo page](https://microsoftedge.github.io/Demos/1DIV/dist/) in a new tab or window, and then select the `<div class="demos">` element in the **Elements** tool.
+
+1. In the **Styles** pane, find the `box-shadow` declaration in the `.demos` CSS rule, and click the **Open shadow editor** (![The shadow editor button icon](./reference-images/shadow-editor-icon.png)) button.
+
+   The **Shadow Editor** opens:
+
+   ![The Shadow Editor](./reference-images/shadow-editor.png)
+
+1. Change the shadow properties, as follows:
+
+   | Property | How to change |
+   |---|---|
+   | **Type** | Select **Outset** or **Inset**.  Only for `box-shadow`. |
+   | **X offset** | Drag the blue dot, or specify a value. |
+   | **Y offset** | Drag the blue dot, or specify a value. |
+   | **Blur** | Drag the slider or specify a value. |
+   | **Spread** | Drag the slider or specify a value.  Only for `box-shadow`. |
+   <!-- todo: explain the meaning of each property, not only how to change the value.  change column heading to:
+   | Property | Description |
+   | Property | Description and how to change |
+   -->
+
+   The changes are applied to the element in the page in real time:
+
+   ![The effect of changes made in the Shadow Editor](./reference-images/shadow-editor-changes.png)
+
+
+<!-- ====================================================================== -->
+## Edit animation and transition timings with the Easing Editor
+
+Use the **Easing Editor** to change the value of the [animation-timing-function](https://developer.mozilla.org/docs/Web/CSS/animation-timing-function) or [transition-timing-function](https://developer.mozilla.org/docs/Web/CSS/transition-timing-function) property on an HTML element:
+
+1. [Select an element](#select-an-element) that has a CSS animation or transition applied.
+
+   For example, open the [animated property demo page](https://microsoftedge.github.io/Demos/devtools-animated-property-issue/) in a new tab or window, and then select the `<div class="spinner">` element in the **Elements** tool.
+
+   The red and green boxes on this demo page are two different CSS animations. Both animations run with CSS, using the `animation` CSS property, and define an `ease-in-out` timing function.
+
+1. In the **Styles** pane, find the `animation` declaration in the `.bad .spinner` CSS rule, and click the **Open cubic bezier editor** (![The cubic bezier editor button icon](./reference-images/easing-editor-icon.png)) button.
+
+   The **Easing Editor** appears:
```

Comment:
```suggestion
   The **Easing Editor** opens:
```
in some cases, i use 'appears', but this kind of editor popup goes well w/ "opens".